js/screens/OnBoarding/OnBoarding.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Text, View, ScrollView, Image } from 'react-native';
+import styles from './styles';
+
+const OnBoarding = () => {
+  return <Text>OnBoard</Text>;
```

Comment:
Please have an actual on boarding screens.

Reply:
hi sid, the on boarding screens will be worked on after the dependencies have been added. do I delete this file instead?